spec/system/page_title_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'rails_helper'
+
+RSpec::Matchers.define :have_page_title do |expected_page_title|
```

Comment:
I think we should pop our two matchers together in `spec/support/matchers` to be tidy.

Reply:
Agreed!